src/coreclr/System.Private.CoreLib/src/Internal/Runtime/InteropServices/InMemoryAssemblyLoader.cs

```diff
@@ -12,9 +12,11 @@ namespace Internal.Runtime.InteropServices
     /// </summary>
     public static class InMemoryAssemblyLoader
     {
+#if TARGET_WINDOWS
```

Comment:
Technically the whole class can be removed from the product on non-Windows - the IJW is not supported on non-Windows. @jkoritzinsky should be able to confirm this.

Reply:
Yes, IJW is unsupported on non-Windows platforms.

Reply:
I think this is being kept around so that this test can build on all platforms: https://github.com/dotnet/runtime/blob/main/src/tests/Interop/IJW/LoadIjwFromModuleHandle/LoadIjwFromModuleHandle.cs#L43

Reply:
That's a bit suspect, since that will always fail. But I'm fine keeping it if @jkoritzinsky thinks it's worth it.

Reply:
It's because in CI we build all the managed tests once on Linux, so the API I think needs to exist for the test to build.

Reply:
Do you build your test agast SPC.dll, that's quite unusual.

Reply:
Yeah, it builds against SPC:https://github.com/dotnet/runtime/blob/fe8ae98d813788605ab034449ec7d313e8baa1ef/src/tests/Interop/IJW/LoadIjwFromModuleHandle/LoadIjwFromModuleHandle.csproj#L4